### ISSUE TITLE
Abnormal results

### ISSUE THREAD

**Issue by domadaaaa** [Hsankesara/VoxelMorph-PyTorch#7]
my running results as followed, why dice and loss  become negative and zero?
>[ 0.43 mins] After 1 epochs, the Average training loss is  tensor(-0.1000, device='cuda:0') and average DICE score is tensor(0.2874, device='cuda:0')
[ 0.24 mins] After 1 epochs, the Average validations loss is  tensor(-1.1711e+11, device='cuda:0') and average DICE score is tensor(0.0014, device='cuda:0')
[ 0.44 mins] After 2 epochs, the Average training loss is  tensor(-0.1628, device='cuda:0') and average DICE score is tensor(0.0002, device='cuda:0')
[ 0.23 mins] After 2 epochs, the Average validations loss is  tensor(0., device='cuda:0') and average DICE score is tensor(0., device='cuda:0')<

**Comment by domadaaaa** [comment]
Is it due to overfitting?

**Comment by domadaaaa** [comment]
I print the reults of registered_image, finding that

> tensor([[[[0., 0., 0.],
          [0., 0., 0.],
          [0., 0., 0.],
          ...,
          [0., 0., 0.],
          [0., 0., 0.],
          [0., 0., 0.]],

         [[0., 0., 0.],
          [0., 0., 0.],
          [0., 0., 0.],
          ...,
          [0., 0., 0.],
          [0., 0., 0.],
          [0., 0., 0.]],

         [[0., 0., 0.],
          [0., 0., 0.],
          [0., 0., 0.],
          ...,
          [0., 0., 0.],
          [0., 0., 0.],
          [0., 0., 0.]],

         ...,

         [[0., 0., 0.],
          [0., 0., 0.],
          [0., 0., 0.],
          ...,
          [0., 0., 0.],
          [0., 0., 0.],
          [0., 0., 0.]],

         [[0., 0., 0.],
          [0., 0., 0.],
          [0., 0., 0.],
          ...,
          [0., 0., 0.],
          [0., 0., 0.],
          [0., 0., 0.]],

         [[0., 0., 0.],
          [0., 0., 0.],
          [0., 0., 0.],
          ...,
          [0., 0., 0.],
          [0., 0., 0.],
          [0., 0., 0.]]]], device='cuda:0', grad_fn=<ReshapeAliasBackward0>)

all elements are zero, I use the data in this program
